### DISCUSSION TITLE
grafana

### DISCUSSION THREAD

**Discussion by saatwika-maker** [grafana/grafana-infinity-datasource#252] [Q&A]
hi sir 
Our project is to display covid data info on grafana dashboard for that we are using json and csv files for scrapping data and mongo db for database later connecting mongodb data to grafana dashboard and displaying it How to connect mongo db and grafana

**Comment by yesoreyeram** [comment]
Have you tried https://grafana.com/grafana/plugins/grafana-mongodb-datasource/?

**Reply by yesoreyeram** [reply]
Note.. mongo db is an enterprise plugin. If you want to connect mongo via infinity datasource, you need to have a rest api layer on top of mongo

**Comment by saatwika-maker** [comment]
Good evening sir
I am Saatwika Matta studying final year b. tech from Amrita School of
engineering .I am currently working on a project where I need to display
covid data on Grafana dashboard by using mongo db .
Can we display mongo db on grafana?
Do we need any enterprise version for connecting mongo db to grafana?
If at all we need an enterprise version is there any account where students
can work on a project?
Can you give detail description about pricing in enterprise version?
